src/main/java/com/example/animalshelter/controller/AnimalController.java

```diff
@@ -0,0 +1,22 @@
+package com.example.animalshelter.controller;
+
+import com.example.animalshelter.model.Animal;
+import com.example.animalshelter.service.AnimalService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController()
```

Comment:
Would do you think about switch it to:
`@RestController
@RequestMapping("/animals")
public class AnimalController {
`

[Here](https://spring.io/guides/tutorials/rest/) are tips about building rest.

Reply:
done